integrations/docker/images/chip-cert-bins/Dockerfile

```diff
@@ -0,0 +1,273 @@
+# Stage 1: Setup dependencies (based on chip-build).
+FROM ubuntu:focal as chip-build-cert
+ARG TARGETPLATFORM
+# BRANCH defines the target branch or tag to build from.
+ARG BRANCH=sve
+VOLUME "/var/source"
```

Comment:
What is the purpose of the Volume here? I don't see it used?

Reply:
It was copied over from chip-build, I think this is what the github actions uses to mount the SDK source. We can remove it.